src/coalesce-vue-vuetify3/src/components/input/c-datetime-picker.vue

```diff
@@ -75,7 +75,16 @@
           :min="min ? startOfDay(min) : undefined"
           :max="max ? endOfDay(max) : undefined"
           v-bind="datePickerProps"
+          :show-adjacent-months="true"
```

Comment:
Putting this after ``v-bind=datePickerProps` makes it impossible for users to customize it. It can already be customized by users via `datePickerProps`, so I don't think this should be enabled by default.